software/speller/data_collection_platform/dcp/cfg/config.py

```diff
@@ -26,8 +29,10 @@ class DevelopmentConfig(Config):
     # better to have it explicitly specified, as sometimes we want to
     # deactivate Flask reloading
     DEBUG = True
-    SQLALCHEMY_DATABASE_URI = os.environ['DEV_DATABASE_URL']
-    broker_url = os.environ["DEV_MESSAGE_QUEUE_URL"]
+    SQLALCHEMY_DATABASE_URI = (f'postgresql+psycopg2://{os.getenv("POSTGRES_USER")}:' + 
```

Comment:
the reason we were using os.environ["something"] is that it would throw an error if not set which is what we want (we don't want to run the app with random settings that we are not sure. Therefore I think it is a good idea to stay with os.environ["something"] over os.getenv() which returns a default value.

Reply:
Inside of the `.env` file, we should set 

`DEV_DATABASE_URL=postgresql://${DEV_DB_USERNAME}:${DEV_DB_PASSWORD}@${DB_HOSTNAME}:${DEV_DB_PORT}/postgres`, 

that way in the config.py we can simply have `SQLALCHEMY_DATABASE_URI = os.environ['DEV_DATABASE_URL']`